app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,10 @@
+class PostsController < ApplicationController
+  def index
+    @user = User.find(params[:user_id])
+    @posts = @user.posts
```

Comment:
- The same issue applies here as pointed out in the UsersController.  To fix this, update the index to 
`def index; end`